lang/en.json

```
@@ -1199,6 +1199,10 @@
   "FIELDS": {
     "duration": {
       "label": "Duration",
+      "concentration": {
+        "label": "Concentration",
+        "hint": "User must maintain concentration while active."
```

I know you don't mean 'user' as in the 'user of the software' case, but more the 'user of the activity' case, but I think 'creature' is a stronger game term here.
```suggestion
        "hint": "Creature must maintain concentration while active."
```